google/ggrc-core
# GGRC-4921 Update permissions for auditor

# Issue description

Auditors (if they have access to the program) can see 'related assessments'.

# Steps to test the changes

1 Create a user with Global Creator role
2 Create a program and audit like Global Admin
3 Add Global Creator user to Audit like Auditor 
4 Create Assessment and add a related assessment to this one.
5 Login like Global Creator
6 Open assessment 
7 See that we do not see "Related Assessments" and "Related Issue" tabs.

# Solution description

Add new Assessment api attr "is_show_related_objs_tabs"

# Sanity checklist


# Migration checklist
-->

# PR Review checklist
